index.js

```diff
@@ -222,8 +228,8 @@ module.exports = postcss.plugin( 'postcss-flexboxfixer', function( opts ) {
     }
 
     return function( css ) {
-        css.walkDecls( /(^background$)/, function( decl ) {
-            if(!/-webkit(-\w*|)-gradient/.test(decl.value)){
+        css.walkDecls(function(decl) {
+            if (!/-webkit-gradient/.test(decl.value)) {
```

Comment:
@hallvors It's not stop stop handling, it is handling every `prop`, not just `background`. like: `background-image`, `list-style-image`, `mask`